plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/RollbackToSnapshotProcedure.java

```diff
@@ -71,10 +68,7 @@ public Procedure get()
 
     public void rollbackToSnapshot(ConnectorSession clientSession, String schema, String table, Long snapshotId)
     {
-        // this line guarantees that classLoader that we stored in the field will be used inside try/catch
-        // as we captured reference to PluginClassLoader during initialization of this class
-        // we can use it now to correctly execute the procedure
-        try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(classLoader)) {
+        try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(getClass().getClassLoader())) {
```

Comment:
Can we do the same cleanup for `io.trino.plugin.iceberg.procedure.RegisterTableProcedure`?

Reply:
sure, go for it!